sklearn/ensemble/_iforest.py

```diff
@@ -369,6 +397,21 @@ def predict(self, X):
         is_inlier : ndarray of shape (n_samples,)
             For each observation, tells whether or not (+1 or -1) it should
             be considered as an inlier according to the fitted model.
+
+        Notes
+        -----
+        The predict method can be parallelized by setting a joblib context. This
+        inherently does NOT use the ``n_jobs`` parameter initialized in the class,
+        which is used during ``fit``. This is because, predict may actually be faster
+        without parallelization for a small number of samples. The user can set the
```

Comment:
```suggestion
        without parallelization for a small number of test-samples of 1000 or less. The user can set the
```

We could mention this explicitly?

Reply:
I am okay with being explicit here.

Reply:
I've updated it to mention that it is better to not parallelize for 1000 samples or less